exposed-core/src/main/kotlin/org/jetbrains/exposed/dao/id/CompositeID.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.exposed.dao.id
+
+import org.jetbrains.exposed.sql.Column
+
+/** Class representing a mapping of each composite primary key column to its stored identity value. */
+class CompositeID(
+    idMapping: Map<out Column<*>, Comparable<*>?>
+) : HashMap<Column<*>, Comparable<*>?>(idMapping), Comparable<CompositeID> {
+    override fun toString(): String = "CompositeID(${entries.joinToString { "${it.key.name}=${it.value}" }})"
+
+    override fun hashCode(): Int = entries.fold(0) { acc, entry ->
```

Comment:
Single entry CompositeId will always have the same value as EntityId of same type